### PR TITLE
 Redundant updating of variables in ActivityLogEntry class #5312 

### LINKED ISSUE #5312: Redundant updating of variables in ActivityLogEntry class
In `ActivityLogEntry` class, we are updating instance variables  `time, servletName, action, role, name, googleId, email, toShow, message, url` in multiple methods.
Implementing internal method that update these instance variables will help reduce redundancy in code and better readablity of code.

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5312

### CODE REVIEW

File: src/main/java/teammates/common/util/ActivityLogEntry.java [hunk 1]
```diff
@@ -75,21 +75,33 @@
     };
     
     /**
+     * updates class variables
```
Comment (damithc) [inline src/main/java/teammates/common/util/ActivityLogEntry.java]: These are not 'class variables'

File: src/main/java/teammates/common/util/ActivityLogEntry.java [hunk 2]
```diff
@@ -113,46 +125,42 @@ public ActivityLogEntry(AppLogLine appLog){
...
-        email = tokens[POSITION_OF_EMAIL];
-        message = tokens[POSITION_OF_MESSAGE];
-        url = tokens[POSITION_OF_URL];
-        
+
+        updateInfo(tokens[POSITION_OF_SERVLETNAME], tokens[POSITION_OF_ACTION], tokens[POSITION_OF_ROLE],
+                                        tokens[POSITION_OF_NAME], googleId = tokens[POSITION_OF_GOOGLEID],
+                                        tokens[POSITION_OF_EMAIL],
+                                        Boolean.parseBoolean(tokens[POSITION_OF_TOSHOW]),
+                                        tokens[POSITION_OF_MESSAGE], tokens[POSITION_OF_URL]);
```
Comment (damithc) [inline src/main/java/teammates/common/util/ActivityLogEntry.java]: Refer to coding standard for indentation guidelines.

**Comment by damithc** [comment]
@tarun14110 While this does reduce duplication, I worry that it reduces readability. In the original code it was easy to see what value is getting assigned to what attribute. The suggested code increases potential errors of inadvertent parameter swapping. I'm inclined to leave the code as it is.

**Comment by tarun14110** [comment]
yeah you are right, it also looks messy to read. Should I close this issue ?

**Comment by damithc** [comment]
Yes. Let me close it for you.

**Pull request closed without merging**